lib/charms/layer/jenkins/api.py

```diff
@@ -136,7 +136,26 @@ def restart(self):
         self.wait()
         unitdata.kv().set("jenkins.last_restart", time.time())
 
+    def quiet_down(self):
+        """
+        Put Jenkins in a Quiet mode, in preparation for a restart.
+        In that mode Jenkins doesn’t start any build.
+        """
+        hookenv.log("Putting Jenkins in Quiet mode.")
+        action = "quietDown"
+        fail_message = "Couldn't put jenkins in Quiet mode"
+        self._execute_action(action, fail_message)
+        hookenv.log("Jenkins is in Quiet mode.")
+        self.wait()
 
+    def cancel_quiet_down(self):
+        """Cancel the quietDown mode"""
```

Comment:
Missing a trailing "." in the docstring.

Reply:
59b9190